src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -146,6 +152,11 @@ private void StartWorkerProcesses(int startIndex, Func<Task> startAction)
                 for (var count = startIndex; count < _maxProcessCount
                     && !_processStartCancellationToken.IsCancellationRequested; count++)
                 {
+                    if (_concurrencyOptions.Value.Enabled && count > 0)
+                    {
+                        // Make sure only one worker is started if concurancy is enabled
```

Comment:
spelling

Reply:
Also, I'd move this check after the start action below and before the delay since we don't need to delay. Consider breaking rather than returning, in case any logic is added after the loop in the future.

Reply:
this check needs to be before the start action as we already started one process